.environ/.env.dev

```diff
@@ -5,9 +5,34 @@
 # Note that although Docker also looks at `.env` files for picking up environmental
 # settings, values present in the docker environment at runtime, e.g. through `docker-compose.yml`,
 # take precedence over these. `docker-compose.yml` has sane defaults. You still need these settings
-# in .env` for the sake of Django settings, though, to be able to run management commands, etc..
+# in `.env` for the sake of Django settings, though, to be able to run management commands, etc..
 #
 DEBUG=true
 SECRET_KEY='dev'
 DATABASE_URL='postgresql://postgres:password@localhost:5432/postgres'
 ENABLE_DEBUG_TOOLBAR=true
+
+# JIRA ########################################################################
+
+ENABLE_JIRA=true
+
+# Set this to your JIRA instance domain.
```

Comment:
Move the comments/explanations to `accounting/settings.py` or to the `README`, to not duplicate them in multiple places? Duplication would eventually result in it being out of date in some places.

Reply:
I actually think the dev environment file is the best place -- that's where people will actually look because that's where they will be editing things. The `README` would be more of a reference for individual vars, and the `accounting/settings.py` comments would be the thing that becomes duplication IMO.

Reply:
Actually I really want to not block this PR on this. I'm going to add a tiny ticket to cleanup the README (as well as add info about prod deployments) next sprint.

Reply:
But this file is committed, so they won't be editing it? And that doesn't address my comment about not duplicating the explanations in multiple places, since you would have multiple environment files?

Reply:
I'm looking at this from a POV of a dev who's diving in for the first time: they see this `.env.dev` file, with a bunch of settings and comments. They copy it over to the root folder as `.env`, and now begin to edit things, and as they edit things they read the comments to understand what should be done to each variable, what the implications are, etc..

I'm confused about what you mean by 'multiple environment files'. Do you mean `.env.test` and `.env`? `.env.test` has no comments, and `.env` is ignored.

Reply:
Yep, that's what I meant by "multiple environment files". If we keep all the comments in `.env.dev`, then that addresses my comment.  Maybe then simply add "# Refer to `.env.dev` for explanations about the variables settings"?

Won't we need a `.env.prod` btw?

Reply:
Yep sure, we can link to `.env.dev`.

`.env.prod` == `.env` basically.